core/interfaces/i_movable.ts

```diff
@@ -16,4 +18,12 @@ export interface IMovable {
    * @returns True if movable.
    */
   isMovable(): boolean;
+
+  /**
+   * Return the coordinates of the top-left corner of this movable object
```

Comment:
Since this changes the API, please change the PR title to "feat:" instead of "fix:"